src/app/workflow/dag/dag.component.html

```diff
@@ -32,6 +32,14 @@
             <input type="radio" name="dagType" [(ngModel)]="dagType" value="cwlviewer">Common Workflow Language Viewer
           </label>
         </div>
+        <div *ngIf="enableWdlViewer && (descriptorType$ | async) === ToolDescriptor.TypeEnum.WDL && (workflow$ | async)?.source_control_provider === 'GITHUB'" style="line-height: 20px;">
+          <label class="radio-inline">
+            <input type="radio" name="dagType" [(ngModel)]="dagType" value="classic">Classic
+          </label>
+          <label class="radio-inline">
+            <input type="radio" name="dagType" [(ngModel)]="dagType" value="wdlviewer">EPAM Pipeline Builder
```

Comment:
I wonder if we should just call it `Pipeline Builder`. Even that is going to be confusing -- I'm not sure that the casual user will know what it is.

Reply:
I'm comfortable with renaming it, since the service being used is an open-source library. Thoughts?